helm/opencensus-collector-app/templates/deployment.yaml

```diff
@@ -29,18 +29,17 @@ spec:
         imagePullPolicy: {{ .Values.image.pullPolicy }}
         resources:
           {{- toYaml .Values.resources | nindent 12 }}
```

Comment:
What is `- toYaml` for? Shouldn't it be just `.Values.resources`?

Reply:
Normally, Values are parsed in to go data structures. To put them back in YAML, you need `toYaml`